spec/ruby/floating_points_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'floating_points'
+
+RSpec.describe FloatingPoints do
+  it 'parses test properly' do
+    r = FloatingPoints.from_file('src/floating_points.bin')
+    
+    delta = 1e-6
+
+    expect(r.single_value).to be_within(delta).of 1.2345
+    expect(r.single_value_be).to be_within(delta).of 1.2345
+    expect(r.double_value).to be_within(delta).of 123.456
+    expect(r.double_value_be).to be_within(delta).of 123.456
```

Comment:
I probably should have used numbers that can be exactly represented as a FP value, my bad. Ruby was being a bit strange on these ones:

     Failure/Error: expect(r.single_value).to eq 1.2345

       expected: 1.2345
            got: 1.2345000505447388

I've not done much Ruby programming so I'm not sure if there's some way to change the behaviour without using a delta.